eskip/eskip.go

```diff
@@ -107,6 +245,15 @@ type Predicate struct {
 	Args []interface{} `json:"args"`
 }
 
+func (p *Predicate) String() string {
+	args, err := getStringArgs(len(p.Args), p.Args)
```

Comment:
getStringArgs is not good here, it used to convert from []interface{} to []string, but the values must be string. Use argsString instead, see: https://github.com/zalando/skipper/pull/1822/files

Reply:
fixed